app/utils/decorator_utils.py

```diff
@@ -0,0 +1,11 @@
+from app.database.models.user import UserModel
+
+def check_mail_confirmation(f):
+
+    def wrapper(user_id,data):
```

Comment:
Informative name rather than generic wrapper.

Reply:
Space between , and data. Please run flake8 before you submit PRs.

Reply:
@ramitsawhney27 flake8 or pylint (I'm thinking of your comment here: https://github.com/systers/mentorship-backend/pull/174#pullrequestreview-198976141)

Reply:
I removed all types of quality errors checked using flake8.

Reply:
Both will definitely inform of the same PEP8 violations, but yes, pylint is better since we've established that.